reflex-algos/src/main/python/mlops/parallelm/mlops/time_capture/plot_functions.py

```diff
@@ -29,15 +29,20 @@ def __init__(self, mtc):
         self._multigraph_df_file = mtc.get_multigraph_df_file()
         self._attribute_names_list = mtc.get_attribute_names_list()
 
-    def line_plot(self, name):
+    def line_plot(self, name, df_name=None):
```

Comment:
so `df_name` is actually a pandas dataframe - maybe a different name?

Reply:
yes